openedx/features/caliper_tracking/tests/current/current_course_enrollment_deactivated.json

```diff
@@ -0,0 +1,24 @@
+{
+  "eventTime": "2018-10-30T08:41:04.621373",
+  "user_id": 6,
+  "referrer": "http://localhost:18000/dashboard",
+  "object": {
+    "course_id": "course-v1:edX+DemoX+Demo_Course",
+    "user_id": 6,
+    "mode": "audit"
+  },
+  "org_id": "edX",
+  "edx_event_type": "edx.course.enrollment.deactivated",
+  "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36",
+  "host": "ffb9adc91cb2",
+  "session": "33065d85817f8fe8c1650e8cf319ccfb",
+  "actor": {
```

Comment:
How are the `actor` and `@context` fields present in the current json? They are caliper specific. Please use the correct file as current input.